lib/github/hooks/issue.ts

```diff
@@ -427,6 +426,99 @@ export const onPullRequestOpened = async (webhooks: Webhooks) => {
   });
 };
 
+export const onPullRequestMerged = async (webhooks: Webhooks) => {
+  webhooks.on(EVENT_TRIGGERS.PULL_REQUEST_CLOSED, async (context) => {
+    const pullRequest = context.payload.pull_request;
+    const repo = context.payload.repository.name;
+    const owner = context.payload.repository.owner.login;
+
+    // Early exit if the pull request was not merged
+    if (!pullRequest.merged) {
+      console.log("Pull request was not merged.");
+      return;
+    }
+
+    const octokit = getOctokitInstance(context.payload.installation?.id!);
+
```

Comment:
let's first check if the PR itself has a points label and if so, award the points. If not, go and search in linked issues.

Reply:
make sure to not repeat the code for awarding points (and if the user isn't found creating one).

we already have that functionality in the /award function. You might have to move it to the utils.ts in the /github folder and then refactor both the onAwardPoints and your new function